best_practices/business-logic.rst

```diff
@@ -10,48 +10,20 @@ your app that's not specific to the framework (e.g. routing and controllers).
 Domain classes, Doctrine entities and regular PHP classes that are used as
 services are good examples of business logic.
 
-For most projects, you should store everything inside the AppBundle.
+For most projects, you should store everything inside the ``src/`` directory.
```

Comment:
`store everything` -> `store all your code` ? Now almost everything is outside `src/` (translations, templates, assets, etc.)

Reply:
Done, thanks!